knockout.typeahead.js

```diff
@@ -1,67 +1,70 @@
-define(["knockout", "jquery", "typeahead", "less/typeahead"], function (ko, $) {
-	ko.bindingHandlers.typeahead = {
-		update: function (element, valueAccessor, allBindings) {
-			// http://stackoverflow.com/a/19366003/1247130 get value to update properly when typeahead choice is selected.
+var ko = require("knockout");
+var $ = require("jquery");
+require("typeahead.js");
+require("typeahead.less");
```

Comment:
Have you tested that these requires work? I think it needs to be `./typeahead` for the less file.